README.md

```diff
@@ -142,4 +142,4 @@ Dia @diashalabi
 
 ## Thanks to
 
-Ryan and it's  stream platform and pyrocms it makes OpenClassify more powerfull. 
+Ryan and it's  stream platform and pyrocms it makes OpenClassify more powerful. 
```

Comment:
Ryan and his stream platform and PyroCMS which makes OpenClassify more powerful.

Reply:
"Ryan and his stream platform PyroCMS which makes OpenClassify more powerful." that might be better. As I know, PyroCMS is Ryan's stream platform.

Reply:
OK :smiley_cat: